class BankAccount{
    public int accountNumber;
    protected String accountHolder;
    private double balance;
    public void setBalance(double b){balance=b;}
    public double getBalance(){return balance;}
}
class SavingsAccount extends BankAccount{
    public void show(){System.out.println(accountNumber+" "+accountHolder);}
}